foundation-extensions/mobile-core/configuration/configuration-api-reference.md

```diff
@@ -189,7 +216,37 @@ MobileCore.updateConfiguration(data);
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+
+### updateConfiguration
+
+ #### Syntax
+
+ ```objectivec
+ @objc(updateConfiguration:)
+ static func updateConfigurationWith(configDict: [String: Any])
+ ```
+
+ #### Example
+
+ #### Objective-C
+
+ ```objectivec
+ NSDictionary *updatedConfig = @{@"global.privacy":@"optedout"};
+ [AEPMobileCore updateConfiguration:updatedConfig];
+ ```
+
+ #### Swift
```

Comment:
Also, it would be nice to document the Swift example first, then Objective-c in all the iOS sections.